src/cards/soul_shard.ts

```diff
@@ -0,0 +1,193 @@
+import * as Unit from '../entity/Unit';
+import * as colors from '../graphics/ui/colors';
+import { CardCategory } from '../types/commonTypes';
+import type Underworld from '../Underworld';
+import { playDefaultSpellSFX } from './cardUtils';
+import { Spell, refundLastSpell } from './index';
+import { CardRarity, probabilityMap } from '../types/commonTypes';
+import { getOrInitModifier } from './util';
+import { distance } from '../jmath/math';
+import { makeManaTrail } from '../graphics/Particles';
+import { Vec2 } from '../jmath/Vec';
+import { addLerpable } from '../lerpList';
+import { soulShardOwnerModifierId } from '../modifierSoulShardOwner';
+
+export const soulShardId = 'Soul Shard';
+const spell: Spell = {
+  card: {
+    id: soulShardId,
+    category: CardCategory.Curses,
+    sfx: 'sacrifice',
+    supportQuantity: false,
+    manaCost: 60,
+    healthCost: 0,
+    expenseScaling: 3,
+    probability: probabilityMap[CardRarity.RARE],
+    thumbnail: 'spellIconSoulShard.png',
+    description: ['spell_soul_shard'],
+    effect: async (state, card, quantity, underworld, prediction) => {
+      // .filter: only target living units
+      const targets = state.targetedUnits.filter(u => u.alive && u.healthMax <= state.casterUnit.healthMax && u != state.casterUnit);
+      if (targets.length) {
+        playDefaultSpellSFX(card, prediction);
+        unitTakeDamageFX(state.casterUnit, underworld, prediction);
+        if (prediction) {
+          // Prediction circles to show affected targets
+          const graphics = globalThis.predictionGraphics;
+          if (graphics) {
+            const lineColor = colors.healthDarkRed;
+            for (let target of targets) {
+              graphics.lineStyle(4, lineColor, 1);
+              // Temp VFX - Draw X
+              graphics.moveTo(target.x + 10, target.y + 10);
+              graphics.lineTo(target.x - 10, target.y - 10);
+              graphics.moveTo(target.x - 10, target.y + 10);
+              graphics.lineTo(target.x + 10, target.y - 10);
```

Comment:
This is cool, how about just the shape of a diamond instead of an "X" and we'll call it good?

Reply:
Sounds good!